app/models/scope.rb

```diff
@@ -1,4 +1,4 @@
 # Scope manages the scope of resources based on roles
 class Scope < ApplicationRecord
-  belongs_to :role
```

Comment:
Don't delete this!!!!!

You can add multiple `belongs_to` so you can have one for `user` as well. But I don't think (in it's current state, it could be changed) scope has anything that points back to a user. Scope relates resources (events, documents, announcements) to roles. The user is then linked in through this.